nuxeo-features/nuxeo-platform-imaging/nuxeo-platform-imaging-core/src/main/java/org/nuxeo/ecm/platform/picture/ImagingComponent.java

```diff
@@ -179,10 +171,8 @@ public ImageInfo getImageInfo(Blob blob) {
             try (CloseableFile cf = blob.getCloseableFile(ext)) {
                 imageInfo = ImageIdentifier.getInfo(cf.getFile().getAbsolutePath());
             }
-        } catch (CommandNotAvailable | CommandException e) {
-            log.error("Failed to get ImageInfo for file " + blob.getFilename(), e);
-        } catch (IOException e) {
-            log.error("Failed to transfer file " + blob.getFilename(), e);
+        } catch (CommandNotAvailable | CommandException | IOException e) {
+            log.error("Failed to get ImageInfo for file {}", blob.getFilename(), e);
```

Comment:
We're not interested in the different cases?

Reply:
I guess the exception message and stack trace will be enough.